Sources/PackageRegistry/RegistryManager.swift

```diff
@@ -185,7 +184,7 @@ public final class RegistryManager {
                         packageLocation: package.description, // FIXME: was originally PackageReference.locationString
                         version: version,
                         revision: nil,
-                        toolsVersion: .currentToolsVersion,
+                        toolsVersion: toolsVersion ?? .currentToolsVersion,
```

Comment:
Can we remove the FIXME on L180 then?

Reply:
this is still not working quite right, and looks like it may require larger change :/

Reply:
ok